data/json/items/vehicle_parts.json

```diff
@@ -1004,6 +1004,21 @@
         "volume":36
     },{
         "type":"GENERIC",
+        "id":"v_drill_item",
+        "symbol":"^",
+        "name":"drill",
+        "name_plural":"drills",
```

Comment:
no need to define plural name in this case since the general rule ```name+"s"``` works just fine for this item.